src/memory/primitives/felt.test.ts

```diff
@@ -0,0 +1,38 @@
+import { test, expect, describe } from 'bun:test';
+import { ConversionError, Felt } from './felt';
+
+describe('felt', () => {
+  test('should initialise a felt correctly and perform good conversions', () => {
+    const felt = new Felt(10n);
+
+    expect(felt.getInner()).toEqual(10n);
+    expect(felt.toNumber()).toEqual(10);
+    expect(felt.toString()).toEqual('10');
+    expect(felt.toHexString()).toEqual('a');
+  });
+
+  test('should fail number conversion when felt inner > JS max number', () => {
+    // twoPowHundred == 2^100
+    const twoPowHundred = 1267650600228229401496703205376n;
```

Comment:
you can just write `2n ** 100n` and remove the comment

Reply:
and actually the test should use Number.MAX_SAFE + 1 for example, and not a random pow (here 100)